dokka-integration-tests/gradle/projects/ui-showcase/build-logic/src/main/kotlin/uitest.dokka-kmp.gradle.kts

```diff
@@ -13,7 +13,7 @@ tasks.withType<AbstractDokkaTask>().configureEach {
     @Language("JSON")
     val dokkaBaseConfiguration = """
     {
-      "footerMessage": "(c) 2024 My custom footer message",
+      "footerMessage": "<div class='kt-footer__license'>Kotlin™ is protected under the <a href='https://kotlinlang.org/foundation/kotlin-foundation.html' data-test='external-link ' target='_blank' rel='noreferrer noopener' class='kt-footer__link'>Kotlin&nbsp;Foundation</a> and licensed under the&nbsp;<a href='https://github.com/JetBrains/kotlin-web-site/blob/master/LICENSE' data-test='external-link ' target='_blank' rel='noreferrer noopener' class='kt-footer__link'>Apache&nbsp;2 license</a>. </div>",
```

Comment:
Not sure about having full markup here. Is there any other place to put this template?

Reply:
Or it's a test snapsthot?

Reply:
> Or it's a test snapsthot?
Yes, this is a more complex example of footer customisation for our testing purposes